api/src/test/java/org/cardanofoundation/rosetta/api/block/service/LedgerBlockServiceImplIntTest.java

```diff
@@ -42,8 +43,7 @@ class LedgerBlockServiceImplIntTest extends IntegrationTest {
   @PersistenceContext
   EntityManager entityManager;
 
-
-  @Test
+  @RepeatedTest(10)
```

Comment:
To make tests concurrent, you can add annotation `@Execution(ExecutionMode.CONCURRENT)` on the class

Reply:
Added